qtfred/src/CheckedDataListModel.h

```diff
@@ -141,21 +141,84 @@ class CheckedDataListModel : public QAbstractListModel
 			return defaultFlags;
 	}
 
-	inline const SCP_unordered_set<T*>& getCheckedData() {
-		return checkeds;
+private:
```

Comment:
In general, I'm happy to comply with this style, but in this case I would prefer leaving this helper method close to the only two places from where it can be meaningfully called. Is this a hard requirement here or would adding a comment clarifying its relation to the two following methods suffice?

Reply:
It is not a hard requirement, just as long as your style is consistent and "logical" within the file.  Additional comments to state the relationship between methods, be it within the doxy of the individual methods or comments to "group" methods together.